astropy 4.3 and it's FITS checks are causing us problems

I have a test that manually creates a FITS file for a test. astopy-4.3 now notes that it's not a "correct" file and throws an error

```
E               OSError: No SIMPLE card found, this file does not appear to be a valid FITS file
```

Of course, we are now getting version 4.3 on the pip installs in our CI run.

This should be an easy change to the test.

--- a/sherpa/astro/io/tests/test_io.py
+++ b/sherpa/astro/io/tests/test_io.py
@@ -150,8 +150,12 @@ def fake_rmf(outfile):
     def hdr(key, value):
         if isinstance(value, str):
             value = "'{}'".format(value)
-        elif isinstance(value, bool):
-            value = 'T' if value else 'F'
+        else:
+            if isinstance(value, bool):
+                value = 'T' if value else 'F'
+
+            # add spacing to make FVERIFY happy
+            value = f"{str(value):>20s}"
 
         out = "{:8s}= {}".format(key, value)
         return out.ljust(80)

